configure.ac

```diff
@@ -872,6 +872,96 @@ AM_CONDITIONAL([OPENTHREAD_ENABLE_RAW_LINK_API], [test "${enable_raw_link_api}"
 AC_DEFINE_UNQUOTED([OPENTHREAD_ENABLE_RAW_LINK_API],[${OPENTHREAD_ENABLE_RAW_LINK_API}],[Define to 1 if you want to enable raw link-layer API])
 
 #
+# Software ACK Timeout Logic
+#
+
+AC_ARG_ENABLE(sw_ack_timeout,
```

Comment:
This is fine as is but I think these may be more suited as config option defines in `openthread-core-default-config.h`. 
I think enabling `raw apis` is a good high-level feature suited for `configure.ac` but the capabilities may be better in the config header file. Thoughts?